solana/pyth2wormhole/client/src/main.rs

```diff
@@ -30,7 +28,10 @@ use {
         attest::P2W_MAX_BATCH_SIZE,
         Pyth2WormholeConfig,
     },
-    pyth2wormhole_client::*,
+    pyth2wormhole_client::{
+        attestation_cfg::SymbolGroup,
+        *,
```

Comment:
I don't know the import best practices here, personally I rather it to the be either * or explicit. (I'm in favor of explicit imports)